code-challenges.js

```diff
@@ -22,11 +57,37 @@ var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
 
+const rem = (arr) => {
+  
+  let newArray = []
+
+  for (let i = 0; i < arr.length; i++) {
+    let remainder = 0
+    if(typeof arr[i] === 'number'){
+    remainder = arr[i] % 3
+    // console.log(remainder)
+    newArray.push(remainder)
+    // return arr[i]
+    }
+  }
+  return newArray
+}
 
+console.log(rem(testingArray1))
+console.log(rem(testingArray2))
 
 
 // --------------------3) Create a function that takes in two arrays as arguments returns one array with no duplicate values.
 
 var testingArray3 = [3, 7, "hi", 10, 3, "hello", 4, "hi"]
 var testingArray4 = [7, "hi", 3, 1, "hi", 4, "hello", 4, 7]
 // Expected output: [ 3, 7, "hi", 10, "hello", 4, 1 ]
+
+
+const dup = (arr1, arr2) => {
+
+  let arr3 = arr2.concat(arr1)
+  let arr4 = arr3.filter((value, index) => arr3.indexOf(value) === index)
+  return arr4
+}
```

Comment:
This is great! I didn't think of using .filter to solve this problem. Looks great :)